### PR TITLE
Portfolio Website: Contact Form

### PR DESCRIPTION
**Pull request by jeanbuhendwa**
# Portfolio Website

In this pull request, I have : 

- Added the contact form section in the index file and styled it using CSS flexbox
- Added the Footer section with the social media links

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,155 +1,246 @@
...
+              />
+              <textarea
+                name="textarea"
+                id="textarea"
+                cols="35"
+                rows="8"
+                placeholder="Enter the Text Here"
+                maxlength="500"
+                required
+              ></textarea>
```
Comment (mukhammadrizooff) [inline index.html:210]: - [ ] [REQUIRED] Good job so far! Please give the correct style to all your `inputs`, I think you forgot to give the following styles: 
- 1. When the user starts typing in the inputs, the `text` color should be black. 
- 2. When the user clicks on Inputs, the color of the input `border`s should be green(`#60C095`) as shown in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=2SxPJKlNwtIkjtSA-0). 
- 3. Please note the distance between the `inputs` and their `size`, they are not the same as the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=2SxPJKlNwtIkjtSA-0).

image

File: index.html [hunk 2]
```diff
@@ -1,155 +1,246 @@
...
+      <section class="contact-section">
+        <div class="contact">
+          <div class="text-contact">
+            <h3>
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </h3>
+          </div>
+          <div class="contact-form">
+            <form action="#" class="contact-form">
```
Comment (mukhammadrizooff) [inline index.html:185]: - [ ] [REQUIRED] Good job so far! Form should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html). action value should be similar to https://formspree.io/f/{form_id} method value equals `POST`. Please implement my suggestion 🙏🏼

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @jeanbuhendwa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @jeanbuhendwa 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors ✔️
 - Good implementations ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**